### DISCUSSION TITLE
Can Playlists be grouped into music and video categories?

### DISCUSSION THREAD

**Discussion by mscroggi** [jellyfin/jellyfin-androidtv#884] [General]
When entering the playlist section of JF, all the playlists are stored together.   
It would be a nice feature if the banner at the top included categories 'Audio' and 'Video' or something.  
Plex does this and I think it is very useful..  I create a lot of different playlists that I like to play in shuffle. 
Its nice to have them separated out into groups.

**Comment by mueslimak3r** [comment]
I'm not sure about the playlist splitting in this app since playlists on the server can store more than one type of media in a playlist.

One option is to create some sorting logic that will call a playlist "music" if it doesn't contain video items, and vice versa.  But this might not be a great design choice since it's going way off spec from the server and other clients.
